src/templates/twoStageSurvey.html

```diff
@@ -20,7 +19,14 @@ <h2 class="bolded-blue">{{pageTitle}}</h2>
         {{/if}}
 
         <div id="individual">
-            <h4 class="bolded-blue" data-i18n="moral-training-task1"></h4>
+            <h4 class="bolded-blue" data-i18n="moral-training-note-desc">###</h4>
```

Comment:
there may not all survey has a description.

please refer to https://github.com/24-S1-2-C-Moral-Decisions/moral-survey/blob/f282006dc1da02ec12506d715fa978439510dc71/src/templates/twoStageSurvey.html#L7-L20